### ISSUE TITLE
Unable to install it on brand new Android 14

### ISSUE THREAD

**Issue by pr2git** [shaxxx/Signalmeter2#6]
Hi,

When a smartphone upgrade to Android 14 and the application is already installed the application is still fully functional.
But somebody who buy a new smartphone with Android 14 and wants to install it, the Play Store report that this application is not compatible with Android 14.

Can you please have a look at it?

Thanks,

Pr2

**Comment by shaxxx** [comment]
Hi there,

Google (and Apple for this matter) are forcing developers with their regular agreements, policies, checks, required info and SDK updates. If you don't accept new license agreement, fail to fill never-ending number of data-forms with background checks, or perhaps fail to recompile your application with their newest SDK (even though you didn't have any code updates) they will take down your app from the store.

The thing is, when moving between phones they will happily copy all the apps you have installed on your previous phone, so you're under impression that it works.

This policy makes sense when application is under continuous development, but makes no sense for application that receive no updates. I had to recompile the app at least 5 times just to keep up with their forced updates and filled enough paperwork to get a loan for a house. For example, with latest requirements the want developers to provide Physical address, full name, ID documents, etc... all in the "interest of the customers".

Well guess what, I have no customers. 
I have 0 interest of going down this Rabbit hole only to allow Google (or Apple) to make money on me.
It' even worse with Apple with yearly developer subscription costs and cost of the hardware.
This is how it basically looks like:

- You want to publish to store? 
- You'll need to use latest tools and sdks
- You want to use latest tools?
- You'll need to use latest OS version to run them
- You want latest OS?
- Sorry, your hardware isn't supported, but good news is for only XXXX€ we'll take care of that
- You got the latest tools and SDKs installed and want to publish to our store?
- We take security seriously, this is pile of paperwork you need to sign, we will validate it and let you know
- You got the paperwork done? 
- Excellent, all you have to do is make a yearly payment, provide 100 screenshots in various resolutions and translations
- You payed and prepared everything?
- Hi there this is some bot rejecting your app because we detected you have button that opens a webpage and on that webpage there is also logo from another store, we don't allow that
- You finally done it? Congratulations, we'll publish your app, but will make it list further down in searches, because, we like payed apps more, they make us REAL money.
- You're still there? Congratulations, let's get back to point one.

TLDR: If I don't find a valid reason/motivation for updating my app (Enigma2 is dead basically) I don't think app will be available from stores anytime soon. But solution is simple, I provide latest build on my server
https://shax.visualstudio.com/SignalMeter/_build/results?buildId=315&view=artifacts&pathAsName=false&type=publishedArtifacts
and you can always download APKs from alternative stores, so no harm done.

Not the answer you wanted, but it's the only one I can give.

Again, thank you for your help with translations.

**Comment by pr2git** [comment]
Thanks for your full explanation, I perfectly understand your point-of-view.

**Comment by shaxxx** [comment]
Just wanted to let you know that app is updated to support latest Android SDK and available for download in Google Play Store.

Once again, thank you for you contribution.
